application/gui/properties_window.py

```diff
@@ -193,6 +197,8 @@ def _rename_item(self, widget=None, data=None):
 
 	def _create_monitor(self):
 		"""Create item monitor"""
+		if hasattr(self, '_monitor') and self._monitor is not None and not self._monitor.is_cancelled():
+			self._monitor.cancel()
```

Comment:
Removed.

On Mon, May 2, 2016 at 12:47 PM, Mladen Mijatov <notifications@github.com>
wrote:

> In application/gui/properties_window.py
> <https://github.com/MeanEYE/Sunflower/pull/230#discussion_r61729014>:
>
> > @@ -193,6 +197,8 @@ def _rename_item(self, widget=None, data=None):
> >
> >  	def _create_monitor(self):
> >  		"""Create item monitor"""
> > +		if hasattr(self, '_monitor') and self._monitor is not None and not self._monitor.is_cancelled():
> > +			self._monitor.cancel()
>
> Please do so I can merge this. We'll need to do extensive testing anyway.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/MeanEYE/Sunflower/pull/230/files/dc4966501ec1bcc599007680f92fcff3e7067830#r61729014>
>